cowbird/api/webhooks/views.py

```diff
@@ -31,7 +31,7 @@ def dispatch(handler_fct):
     if not handlers:
         LOGGER.warning("No handlers matched for dispatch of event [%s].", event_name)
     if exceptions:
-        raise Exception(exceptions)
+        raise Exception(exceptions)  # pylint: disable=W0719
```

Comment:
I tend to agree we `pylint` here. It would be better to have some kind of `WebhookDispatchError` since it get "handled" minimally.